pkg/metainfo/kvmetainfo/temputils.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package kvmetainfo
+
+import (
+	"github.com/vivint/infectious"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/storage/segments"
+	"storj.io/storj/pkg/storage/streams"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/uplink/metainfo"
+)
+
+var (
+	// Error is the errs class of SetupProject
+	Error = errs.Class("SetupProject error")
+)
+
+// SetupProject creates a project with temporary values until we can figure out how to bypass encryption related setup
+func SetupProject(m metainfo.Client) (*Project, error) {
+	whoCares := 1 // TODO: find a better way to do this
+	fc, err := infectious.NewFEC(whoCares, whoCares)
+	if err != nil {
+		return nil, Error.New("failed to create erasure coding client: %v", err)
+	}
+	rs, err := eestream.NewRedundancyStrategy(eestream.NewRSScheme(fc, whoCares), whoCares, whoCares)
+	if err != nil {
+		return nil, Error.New("failed to create redundancy strategy: %v", err)
+	}
+	maxBucketMetaSize := 10 * memory.MiB
+	segment := segments.NewSegmentStore(m, nil, rs, maxBucketMetaSize.Int(), maxBucketMetaSize.Int64())
+
+	// volatile warning: we're setting an encryption key of all zeros when one isn't provided.
```

Comment:
so does this mean that the satellite will have to decrypt bucket metadata with a key of all zeros to display bucket info in the web UI?

Reply:
All zeros is essentially a nil encryption key, so the bucket metadata isn't supposed to be encrypted